### ISSUE TITLE
Useref should ignore the comments in blocks

### ISSUE THREAD

**Issue by stakach** [digisfera/useref#44]
```html
<!-- build:js(.) scripts/vendor.js -->
    <!--script src="bower_components/jquery/dist/jquery.js"></script-->
    <!-- <script src="bower_components/jquery-ui/jquery-ui.js"></script> -->
    <script src="bower_components/angular/angular.js"></script>
    <script src="bower_components/angular-ui-router/release/angular-ui-router.js"></script>
<!-- endbuild -->
```

In development neither jquery or jquery ui are loaded
However useref will still add these files to the build

**Comment by stakach** [comment]
Original issue jonkemp/gulp-useref#100

**Comment by manuelcabral** [comment]
Closed by #47

**Comment by MarkyMarkNo1** [comment]
Ho @manuelcabral, I'm recognizing this issue - useref _should not_ aplly to commented assets _but does_ - is still present?

https://github.com/digisfera/useref/issues/44#issue-78280133 

> ```
> 
>     
>     
>     <script src="bower_components/angular/angular.js"></script>
>     <script src="bower_components/angular-ui-router/release/angular-ui-router.js"></script>
> 
> 
> ```
> In development neither jquery or jquery ui are loaded
> However useref will still add these files to the build

Thank you!
